chainerx_cc/chainerx/native/native_float16_test.cc

```diff
@@ -0,0 +1,168 @@
+#include "chainerx/native/native_float16.h"
+
+#include <cmath>
+#include <limits>
+#include <vector>
+
+#include <gtest/gtest.h>
+
+namespace chainerx {
+namespace native {
+namespace {
+
+void CheckNear(double expected, double actual, double tol) {
+    if (std::isinf(expected)) {
+        EXPECT_TRUE(std::isinf(actual));
+    } else if (std::isnan(expected)) {
+        EXPECT_TRUE(std::isnan(actual));
+    } else if (expected == 0.0) {
+        EXPECT_EQ(actual, 0.0);
+    } else {
+        if (-1.0 < expected && expected < 1.0) {
+            EXPECT_NEAR(expected, actual, tol);
+        } else {
+            EXPECT_NEAR(actual / expected, 1.0, tol);
+        }
+    }
+}
+
+void CheckFloatSixteenEqual(double x, double tol) {
```

Comment:
Explain what `x` is supposed to be, because the type `double` does not match the function name, which says about `float16`.